tests/components/openuv/test_config_flow.py

```diff
@@ -40,8 +41,8 @@
     flow = config_flow.OpenUvFlowHandler()
     flow.hass = hass
 
-    with patch('pyopenuv.util.validate_api_key',
-               return_value=mock_coro(False)):
+    with patch.object(pyopenuv.Client, 'uv_index',
```

Comment:
Extract this into a pytest fixture that takes another fixture as argument which is the coroutine return value.

Reply:
I'm confused – can you show an example of what you're asking?